lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/TransactionReceipt.kt

```diff
@@ -83,7 +131,16 @@ data class GatewayTransactionReceipt(
     override val blockNumber: Int? = null,
 
     @JsonNames("status")
-    override val status: TransactionStatus = TransactionStatus.UNKNOWN,
+    val status: TransactionStatus = TransactionStatus.UNKNOWN,
+
+    @JsonNames("execution_status")
+    override val executionStatus: TransactionExecutionStatus,
+
+    @JsonNames("finality_status")
+    override val finalityStatus: TransactionFinalityStatus,
+
+    @JsonNames("revert_error", "revert_reason")
```

Comment:
Does gateway still return `revert_reason`?

Reply:
Actually, both Gateway and JsonRpc providers seem to return `revert_error`. While in the 0.4.0 spec, there's no mention of `revert_error` and only `revert_error` is used.

For now I think we can't go wrong with accepting both.